src/ray/raylet/node_manager.cc

```diff
@@ -1715,6 +1781,9 @@ void NodeManager::HandleCommitBundleResources(
   RAY_LOG(DEBUG) << "Request to commit resources for bundles: "
                  << GetDebugStringForBundles(bundle_specs);
   placement_group_resource_manager_->CommitBundles(bundle_specs);
+  if (RayConfig::instance().use_ray_syncer()) {
```

Comment:
hmm should we call BroadcastMessage on cluster resource manager/cluster resource scheduler changes?

Reply:
I'm open to that. But my concern here is that in that way, resource manager/cluster resource scheduler is going to have a reference to ray_syncer_ which makes the structure not that clear. (implicit circular reference).

Here what we'd like is to trigger a broadcast immediately for pg.